<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameModeDialog</class>
 <widget class="QWidget" name="GameModeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="button_help">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>aide</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>160</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="button_normal">
      <property name="text">
       <string>version normal</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_allonge">
      <property name="text">
       <string>version allong√©</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="quit_button">
      <property name="text">
       <string>quit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
